src/Qowaiv/Globalization/CountryConstants.cs

```diff
@@ -92,7 +92,7 @@ namespace Qowaiv.Globalization;
     /// <summary>Describes the country Bolivia (BO).</summary>
     public static readonly Country BO = new("BO");
 
-    /// <summary>Describes the country Bonaire (BQ).</summary>
+    /// <summary>Describes the country Caribbean Netherlands (BQ).</summary>
```

Comment:
Is this right? Earlier it was the combination of Bonaire, Sint Eustasius and Saba

Reply:
Caribbean Netherlands is the combination of Bonaire, Sint Eustasius and Saba, so yes. Unless you have a source that explains that is wrong?!

Reply:
It is more that the Caribbean Netherlands is used to describe the collective of those three, should it then also be used to describe one of the three?

Reply:
BQ is for Bonaire, Sint Eustasius and Saba, not just Bonaire.